source/content/guides/build-tools/03-pr-workflow.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Build Tools
+subtitle: Pull Request Workflow
+description: In step three of the Build Tools guide, learn how to use pull requests as part of your workflow.
+buildtools: true
+anchorid: pr-workflow
+generator: pagination
+layout: guide
+type: guide
+pagination:
+    provider: data.buildtoolspages
+use:
+    - buildtoolspages
+permalink: docs/guides/build-tools/pr-workflow/
+nexturl: guides/build-tools/configure/
+previousurl: guides/build-tools/create-project/
+editpath: build-tools/03-pr-workflow.md
+image: buildToolsGuide-thumb
+---
+
+## Pull Request Introduction
+This section demonstrates the Build Tools project workflow by making a code change on a Git feature branch and opening a Pull Request to accept that change into the `master` branch.
+
+As a reminder, you will be working with the code repository hosted on <GitProvider />. <CIProvider /> will then build the full site artifact and deploy it to the project's Pantheon site.
+
+Branches with an associated pull request are built and deployed to a [Pantheon multidev environment](https://pantheon.io/docs/multidev), with the pull request number being used to construct the multidev name. For example, pull request `12` will be deployed to a Pantheon multidev named `pr-12`.
+
+The `master` branch is automatically built and deployed to the Pantheon `dev` environment. This happens both when you commit code directly to the `master` branch and when a pull request is merged into the `master` branch.
+
+![Continuous delivery diagram](../../../images/pr-workflow/github-circle-pantheon.png)
+
+Deployments to the `test` and `live` environments on Pantheon must still be done manually, either in the dashboard or via Terminus. For this reason, Continuous Delivery <Popover title="Continuous Delivery" content="Continuous Delivery (CD) is the practice of automatically deploying code all the way to production, without human intervention. This requires a consistently clear deployment pipeline from development to production. That is to say, an application must be able to deploy code to production at any given time regardless of current work in progress." /> is not enabled.
+
+## Create a Pull Request
+
+1. From your <GitProvider /> project page, click on the `config` directory. Select the file named `system.site.yml` and click <span class="glyphicon glyphicon-pencil"></span> to open an editor:
```

Comment:
I worry that this example here is very GitHub specific. Maybe we should label it as such, include a note that your exact workflow may vary with alternate providers, or include examples of the other providers as well?

Reply:
Done! Labelled as Github + Circle CI instructions and added a note. 

Will see if Edward has other ideas about how to manage this. Thank you!